snow/engine/snowman/bootstrap/storage.go

```diff
@@ -133,6 +134,12 @@ func execute(
 	tree *interval.Tree,
 	lastAcceptedHeight uint64,
 ) error {
+
+	if tree.Len() > minBlocksToCompact {
+		log("compacting before iteration...")
```

Comment:
Super minor nit just to align the log a bit more with the following `executing blocks` log
```suggestion
		log("compacting database before executing blocks...")
```